include/xrpl/protocol/TxFlags.h

```diff
@@ -185,6 +185,14 @@ constexpr std::uint32_t tfDepositMask = ~(tfUniversal | tfDepositSubTx);
 constexpr std::uint32_t tfClearAccountCreateAmount     = 0x00010000;
 constexpr std::uint32_t tfBridgeModifyMask = ~(tfUniversal | tfClearAccountCreateAmount);
 
+// Batch Flags
+constexpr std::uint32_t tfAllOrNothing                 = 0x00000001;
+constexpr std::uint32_t tfOnlyOne                      = 0x00000002;
+constexpr std::uint32_t tfUntilFailure                 = 0x00000004;
+constexpr std::uint32_t tfIndependent                 = 0x00000008;
```

Comment:
These flags should leave the smallest 2 bytes empty - those bits are reserved. https://xrpl.org/docs/references/protocol/transactions/common-fields#flag-ranges